scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/factories/ValidationUtil.java

```diff
@@ -29,24 +29,20 @@
 import static org.ow2.proactive.scheduler.common.job.factories.XMLTags.TASK;
 import static org.ow2.proactive.scheduler.common.job.factories.XMLTags.TASK_FLOW;
 
+import java.io.ByteArrayOutputStream;
 import java.io.File;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.io.File'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)